tests/util.py

```diff
@@ -160,3 +160,18 @@ def generate_path(dirname, filename, encoding=None):
     if encoding:
         path = path.encode(encoding)
     return path
+
+
+def sanlock_is_running():
+    exe_file = "./src/sanlock"
+    pid_file = "/tmp/sanlock/sanlock.pid"
+    try:
+        with open(pid_file, "r") as f:
+            for line in f:
+                pid = line.strip()
+                if os.path.samefile(os.readlink("/proc/"+str(pid)+"/exe"), exe_file):
+                    return True
+    except OSError:
+        return False
+
+    return False
```

Comment:
Would be little nicer and more correct as:

    proc_exec = "/proc/{}/exec".format(pid)

    try:
        executable = os.readlink(proc_exe)
        return os.path.samefile(executable, SANLOCK)
    except EnvironmentError as e:
        if e.errno != errno.ENOENT:
            raise

    return False